src/components/pages/TopNav.astro

```diff
@@ -1,165 +1,297 @@
 ---
-import { getLangFromUrl, useTranslations } from '../../i18n/utils';
+import { getLangFromUrl, useTranslations } from '../../i18n/utils'
+import NavToolsDropdown from './NavToolsDropdown.astro'
 
-const lang = getLangFromUrl(Astro.url);
-const t = useTranslations(lang);
-const homePageUrl = lang === 'en' ? '/' : `/${lang}/`;
+const lang = getLangFromUrl(Astro.url)
+const t = useTranslations(lang)
+const homePageUrl = lang === 'en' ? '/' : `/${lang}/`
 ---
+
 <header>
```

Comment:
I was wondering if on mobile, we should add some effect so the text that's on page doesn't add noise to the the area above nav.



| `backdrop-filter: blur(2px);` | current |
| --- | --- |
| <img width="376" height="402" alt="image" src="https://github.com/user-attachments/assets/03162486-b47c-4c7a-8dcb-f018de898ad9" /> | <img width="376" height="402" alt="image" src="https://github.com/user-attachments/assets/5de6f8e6-06d8-4190-a2b0-91cb11a432e0" /> |
| <img width="376" height="402" alt="image" src="https://github.com/user-attachments/assets/b12d058b-4941-4e96-8295-597ca3ec3027" /> | <img width="376" height="402" alt="image" src="https://github.com/user-attachments/assets/20e3faad-a196-4e0d-bf94-6680be5ba1b4" /> |

Reply:
Decision to keep as it is now.